python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py

```diff
@@ -107,7 +107,7 @@
 graph.add(cxy_ex)
 graph.setLegends(["$Q_2$ kernel smoothing", "$Q_2$ exact"])
 graph.setLegendPosition("lower right")
-graph.setColors(["red", "blue"])
+#graph.setColors(["red", "blue"])
```

Comment:
Unsure about this deletion, were the colors not specified in order to override the default in this particular case ?